src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/MessagingProvider.cs

```diff
@@ -108,5 +153,19 @@ private MessageSender GetOrAddMessageSender(string entityPath, string connection
             string cacheKey = $"{entityPath}-{connectionString}";
             return _messageSenderCache.GetOrAdd(cacheKey, new MessageSender(connectionString, entityPath));
         }
+
+        private ClientEntity GetOrAddClientEntity(string entityPath, string connectionString)
+        {
+            string cacheKey = $"{entityPath}-{connectionString}";
+            string[] arr = entityPath.Split('/');
+            if (arr.Length > 1)
+            {
+                return _clientEntityCache.GetOrAdd(cacheKey, new SubscriptionClient(connectionString, arr[0], arr[2]));
```

Comment:
Can you add a comment for what's happening here? I see you split it and then take [0] and [2] -- what about [1]? Maybe having a sample string that you're looking for would make it more clear.

Reply:
Done.